source/_components/sensor.trafikverket_weatherstation.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "Trafikverket WeatherStation"
+description: "Instructions how to integrate Trafikverket WeatherStation within Home Assistant."
+date: 2018-02-01 12:06
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: trafikverket.png
+ha_category: Sensor
+ha_release: 0.63.0
+---
+
+Showing weather information for air and road temperature provided by Trafikverket in Sweden. 
+
+Usage: Get weather data if you live near a weather station, just want to have the current temperature, needing the current temperature to know when your car heating system should start or when to leave for work due to queues (because the road is frozen).
+
+##### {% linkable_title Configuration %}
+To enable this sensor, add the following lines to your `configuration.yaml`.
+
+```yaml
+sensor:
+  - platform: trafikverket_weatherstations
+    name: Trafikverket Road WeatherStation Kungälv
+    api_key: eXXcbXXXacXXXXc39XX3aXXX4aXX46XX
+    station: Kungälv
+    type: road
+```
+
+Configuration variables:
+
+- **name** (*Required*): Unique name of the device in the frontend.
+- **api_key** (*Required*): API key from Trafikverket
+- **station** (*Required*): Name of the weather station
+- **type** (*Required*): Defines which temperature you want (`air` or `road`)
+- **scan_interval** (*Optional*): How frequently to query for new data (in seconds). Defaults to 300 seconds (5 minutes)
```

Comment:
Configuration needs to be defined in a pre-defined manner, see https://home-assistant.io/developers/documentation/standards/#component-and-platform-pages and https://github.com/home-assistant/home-assistant.github.io/blob/next/source/_components/sensor.temper.markdown for an example.

Reply:
Why is this needed? Some components doesn't have it, like the Roth Touchline. Is it different requirements? I can change it, just curious to learn.

Reply:
I think it is to keep the documentation parseable (to allow consistent UI representations among components I suppose, in case the current one gets changed, it looks already different than those "old-style" ones.), but I don't know for sure either. I just saw that a while ago and updated my own PR accordingly.

Reply:
Another use-case I can think of is notifying the user / developer that the configuration format has changed, if it differs from the source code. This could be really helpful to avoid stale documentation, and should probably also be done on service calls (not relevant for this PR though).

Reply:
Updated to use `{% configuration %}`, please review.